ETR01SDK-465: Mention deletion of R_MEM_DATA_SIZE_MAX in changelog for 3.0.0 (#402)
tropicsquare/libtropic: CHANGELOG.md

    - Signature verification was removed from the HW wallet example.
    - Users should verify the signatures themselves e.g., using functions provided by their crypto library.
- `lt_update_mode` function.

> - `R_MEM_DATA_SIZE_MAX` macro, as the value is different for some Application FW versions. Use the correct version of [User API or datasheet](https://github.com/tropicsquare/tropic01?tab=readme-ov-file#datasheet-and-user-api) (based on the used Application FW) as the source of truth for these kind of constants.

Reviewer: ```suggestion
- `R_MEM_DATA_SIZE_MAX` macro because its value differs between Application FW versions. Use the appropriate [User API or datasheet](https://github.com/tropicsquare/tropic01?tab=readme-ov-file#datasheet-and-user-api) (based on the Application FW in use) as the source of truth for such constants.
```
